packages/components/src/components/wallet-card/wallet-card.tsx

```diff
@@ -52,11 +52,15 @@ const WalletCard: React.FC<React.PropsWithChildren<TWalletCardProps>> = ({
                     <div className={`wallet-card__content wallet-card__content--${size}`}>
                         <div className='wallet-card__top-wrapper'>
                             <IconComponent />
-                            <Badge
-                                custom_color='var(--text-prominent)'
-                                label={wallet.jurisdiction_title}
-                                type='bordered'
-                            />
+                            {wallet.jurisdiction_title === 'virtual' ? (
+                                <Badge label={'demo'} type='contained' background_color='blue' />
+                            ) : (
+                                <Badge
+                                    custom_color='var(--text-prominent)'
+                                    label={wallet.jurisdiction_title}
+                                    type='bordered'
+                                />
+                            )}
```

Comment:
```suggestion

<Badge
    label={wallet.jurisdiction_title === 'virtual' ? 'demo' : wallet.jurisdiction_title}
    type={wallet.jurisdiction_title === 'virtual' ? 'contained' : 'bordered'}
    background_color={wallet.jurisdiction_title === 'virtual' ? 'blue' : undefined}
    custom_color={wallet.jurisdiction_title === 'virtual' ? undefined : 'var(--text-prominent)'}
/>

```

Reply:
@aizad-deriv I can't do like this because `background_color` can be just 'blue' | 'orange' | 'red' | 'gray' and this property applied only when `type` === `bordered`.

Reply:
Just out of curiousity is there a default color? If no, then I think it is fine. Other than that it looks good 😃👍

Reply:
@aizad-deriv when `type` === `'contained'` then background must be `'blue' | 'orange' | 'red' | 'gray'`, but of `type` === `'bordered'` then `background` will not be applied and can be skipped

Reply:
Thanks for the explanation I understand now 🙏🏻